hicommonwealth/commonwealth
# Migrated env feature flags to openfeature

## Link to Issue
Closes: #6701

## Description of Changes
- Migrates old feature flag system to use openfeature.

## Test Plan
With the .env:
```
FLAG_COMMUNITY_STAKE=true
FLAG_NEW_ADMIN_ONBOARDING=true
```

- Check community stakes by creating a staked community with the sepolia chain node.
- Create community and check that the new admin onboarding section appears
